src/openms/source/FORMAT/MzQCFile.cpp

```diff
@@ -54,14 +57,15 @@ using namespace std;
 
 namespace OpenMS
 { 
-
   void MzQCFile::store(const String& input_file,
                        const String& output_file,
                        const MSExperiment& exp,
```

Comment:
why do we have input_file and exp? Shouldn't exp be enough?

Reply:
we need to store the input file location and name